.circleci/config.yml

```
@@ -33,6 +33,33 @@ defaults: &defaults
   working_directory: ~/repo
 
 jobs:
+  build_elixir_1_10_otp_23:
+    docker:
+      - image: erlang:23.0
```

```suggestion
      - image: erlang:23.0.1
```
This avoids a random compiler hang.